### PR TITLE
increase mod version

### PR DESCRIPTION
**Pull request by LooJee**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

### CODE REVIEW

File: pkg/picker/config.go [hunk 1]
```diff
@@ -7,13 +7,14 @@ import (
...
-// 		Function
-// 		CheckOnPost
+//
+//			ConcatItems > Indices
+//			Nullable
+//			DefaultValue
+//	   NullValue, if set to null, subsequent conversions will be skipped.
+//			Type
+//			Function
+//			CheckOnPost
```
Comment (veezhang) [inline pkg/picker/config.go:17]: Please revert the changes here, thank you!

Comment (LooJee) [reply pkg/picker/config.go]: sorry, I don't know why my format tool did this. I have fixed it.

**Review by veezhang — APPROVED** [review]
LGTM
